test/SourceKit/CodeComplete/complete_typerelation.swift.convertible.response

```diff
@@ -26,39 +26,39 @@
     },
     {
       key.kind: source.lang.swift.decl.function.method.instance,
-      key.name: "intanceReturnMyEnum(:)",
-      key.sourcetext: "intanceReturnMyEnum(<#T##self: MyEnum##MyEnum#>)",
-      key.description: "intanceReturnMyEnum(self: MyEnum)",
+      key.name: "instanceReturnMyEnum(:)",
+      key.sourcetext: "instanceReturnMyEnum(<#T##self: MyEnum##MyEnum#>)",
+      key.description: "instanceReturnMyEnum(self: MyEnum)",
       key.typename: "() -> MyEnum",
       key.context: source.codecompletion.context.thisclass,
       key.typerelation: source.codecompletion.typerelation.unrelated,
       key.num_bytes_to_erase: 0,
-      key.associated_usrs: "s:21complete_typerelation6MyEnumO013intanceReturncD0ACyF",
+      key.associated_usrs: "s:21complete_typerelation6MyEnumO013instanceReturncD0ACyF",
```

Comment:
@tshortli: My guess is that the mangling here is wrong....

Presumably I want:
```
s:21complete_typerelation6MyEnumO014instanceReturncD0ACyF
```